apps/bfd-pipeline/bfd-pipeline-rif-to-rda/src/main/java/gov/cms/bfd/pipeline/bridge/RDABridge.java

```diff
@@ -0,0 +1,233 @@
+
+@Slf4j
+public class RDABridge {
+
+  private enum SourceType {
+    FISS,
+    MCS
+  }
+
+  /**
+   * Handles translation of a CLI execution, validating and pulling arguments to then invoke the
+   * underlying application code with.
+   *
+   * @param args Array of the command line arguments.
+   */
+  public static void main(String[] args) {
+    try {
+      // Build CLI commands for executing this via CLI
+      CommandArguments arguments = new CommandArguments("run_bridge.sh");
+      arguments
+          .register()
+          .flag("o")
+          .label("-o [outputDir]")
+          .description("The directory where the output files will be written to.")
+          .argumentCount(1);
+      arguments
+          .register()
+          .flag("i")
+          .label("-i [icnStart]")
+          .description("The starting ICN value to use for generated MCS claims.")
+          .argumentCount(1);
+      arguments
+          .register()
+          .flag("d")
+          .label("-d [dcnStart]")
+          .description("The starting DCN value to use for generated FISS claims.")
+          .argumentCount(1);
+      arguments
+          .register()
+          .flag("m")
+          .label("-m [icnPattern]")
+          .description("The ICN pattern to use for generated ICNs in MCS claims.")
+          .argumentCount(1);
+      arguments
+          .register()
+          .flag("f")
+          .label("-f [dcnPattern]")
+          .description("The DCN pattern to use for generated DCNs in FISS claims.")
+          .argumentCount(1);
+      arguments
+          .register()
+          .argument()
+          .label("inputDir")
+          .description("The directory containing the files to read from.");
+      arguments.addAll(args);
+
+      if (arguments.hasArgs()) {
+        String rifRootDir =
+            arguments
+                .getArg(0)
+                .orElseThrow(
+                    () ->
+                        new IllegalArgumentException(
+                            "No root directory for RIF files provided.\n" + arguments.getUsage()));
+        String outputDir = arguments.getFlagValue("o").orElse(null);
+        long icnStart = arguments.getFlagLongValue("i").orElse(0L);
+        long dcnStart = arguments.getFlagLongValue("d").orElse(0L);
+
+        new RDABridge().run(new GenConfig(rifRootDir, outputDir, icnStart, dcnStart, null, null));
+      } else {
+        log.error("Invalid execution\n" + arguments.getUsage());
+      }
+    } catch (IOException e) {
+      log.error("Failed to execute", e);
+      System.exit(1);
+    }
+  }
+
+  /**
+   * Reads all relevant source files, executing task logic for each claim found.
+   *
+   * @param config The configurations to use when generating the RDA data.
+   * @throws IOException If there was an issue accessing any of the files.
+   */
+  public void run(GenConfig config) throws IOException {
+    String[] fissSources = {"inpatient", "outpatient", "home", "hospice", "snf"};
```

Comment:
fine for defaults, but can we add an option to override this either via command line arguments or a configuration file